osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySpinner.cs

```diff
@@ -16,6 +16,13 @@ namespace osu.Game.Rulesets.Osu.Skinning.Legacy
 {
     public abstract class LegacySpinner : CompositeDrawable
     {
+        /// <summary>
+        /// An offset that simulates stable's spinner top offset, can be used with <see cref="LegacyCoordinatesContainer"/>
+        /// for positioning some legacy spinner components perfectly as in stable.
+        /// (e.g. 'spin' sprite, 'clear' sprite, metre in old-style spinners)
+        /// </summary>
+        public static readonly float SPINNER_TOP_OFFSET = (float)Math.Ceiling(45f * SPRITE_SCALE);
```

Comment:
```suggestion
        public static readonly float SPINNER_TOP_OFFSET = MathF.Ceiling(45f * SPRITE_SCALE);
```

Reply:
is there an analyser or something that can do this automatically?

Reply:
Not that I know of